frontend/src/features/public-form/components/PublicSwitchEnvMessage.tsx

```diff
@@ -10,14 +10,25 @@ export const PublicSwitchEnvMessage = (): JSX.Element => {
   const { publicSwitchEnvMutation } = useEnvMutations()
 
   return (
-    <InlineMessage variant="warning" mb="1.5rem" mt={{ base: '2rem', md: '0' }}>
-      <Text>
-        You’re filling this form on the new FormSG. If you have trouble
-        submitting,{' '}
```

Comment:
Just checking: explicit space wasn't needed?

Reply:
oops that was my bad! I'll change it in my [react-to-angular PR](https://github.com/opengovsg/FormSG/pull/4352) since it should apply to both `PublicSwitchEnvMessage` and `AdminSwitchEnvMessage` :)